addons/laser/functions/fnc_keyLaserCodeChange.sqf

```diff
@@ -40,8 +40,9 @@ if (isNull (ACE_controlledUAV param [0, objNull])) then {
 };
 
 TRACE_2("",_currentShooter,_currentWeapon);
-if (((getNumber (configFile >> "CfgWeapons" >> _currentWeapon >> "laser")) == 0) &&
-        {(getNumber (configFile >> "CfgWeapons" >> _currentWeapon >> QGVAR(canSelect))) == 0}) exitWith {false};
+if (((getNumber (configFile >> "CfgWeapons" >> _currentWeapon >> "laser")) == 0) && 
```

Comment:
```suggestion
private _currentWeaponCfg = configFile >> "CfgWeapons" >> _currentWeapon;
if ((getNumber (_currentWeaponCfg  >> "laser") == 0) && 
```

Save some repeated lookups. Same in the lower conditions.
Not sure if worth, but IMO also better readability through shother lines.